src/logging.cpp

```diff
@@ -7,7 +7,6 @@
 #include <logging.h>
 #include <util/threadnames.h>
 #include <util/string.h>
-#include <util/time.h>
```

Comment:
Why are you removing this include when it is needed?

Reply:
According to the dev notes and iwyu it will need to be included in both files if both files use it.